plugin/evm/limitorders/abis.go

```diff
@@ -1,1057 +1,1391 @@
-        }
-      ],
-      "name": "OrdersMatched",
-      "type": "event"
-    },
-    {
-      "inputs": [
-        {
-          "components": [
-            {
-              "internalType": "uint256",
-              "name": "ammIndex",
-              "type": "uint256"
-            },
-            {
-              "internalType": "address",
-              "name": "trader",
-              "type": "address"
-            },
-            {
-              "internalType": "int256",
-              "name": "baseAssetQuantity",
-              "type": "int256"
-            },
-            {
-              "internalType": "uint256",
-              "name": "price",
-              "type": "uint256"
-            },
-            {
-              "internalType": "uint256",
-              "name": "salt",
-              "type": "uint256"
-            }
-          ],
-          "internalType": "struct IOrderBook.Order[2]",
-          "name": "orders",
-          "type": "tuple[2]"
-        },
-        {
-          "internalType": "bytes[2]",
-          "name": "signatures",
-          "type": "bytes[2]"
-        },
-        {
-          "internalType": "int256",
-          "name": "fillAmount",
-          "type": "int256"
-        }
-      ],
-      "name": "executeMatchedOrders",
-      "outputs": [],
-      "stateMutability": "nonpayable",
-      "type": "function"
-    },
-    {
-      "inputs": [],
-      "name": "getLastTradePrices",
-      "outputs": [
-        {
-          "internalType": "uint256[]",
-          "name": "lastTradePrices",
-          "type": "uint256[]"
-        }
-      ],
-      "stateMutability": "view",
-      "type": "function"
-    },
-    {
-      "inputs": [
-        {
-          "internalType": "address",
-          "name": "trader",
-          "type": "address"
-        },
-        {
-          "components": [
-            {
-              "internalType": "uint256",
-              "name": "ammIndex",
-              "type": "uint256"
-            },
-            {
-              "internalType": "address",
-              "name": "trader",
-              "type": "address"
-            },
-            {
-              "internalType": "int256",
-              "name": "baseAssetQuantity",
-              "type": "int256"
-            },
-            {
-              "internalType": "uint256",
-              "name": "price",
-              "type": "uint256"
-            },
-            {
-              "internalType": "uint256",
-              "name": "salt",
-              "type": "uint256"
-            }
-          ],
-          "internalType": "struct IOrderBook.Order",
-          "name": "order",
-          "type": "tuple"
-        },
-        {
-          "internalType": "bytes",
-          "name": "signature",
-          "type": "bytes"
-        },
-        {
-          "internalType": "int256",
-          "name": "toLiquidate",
-          "type": "int256"
-        }
-      ],
-      "name": "liquidateAndExecuteOrder",
```

Comment:
why did we remove this function from abi ?

Reply:
good catch. it was missing from the interface in the protocol repo. added now.